docs/guide/en/dropdown.md

```diff
@@ -0,0 +1,215 @@
+# Dropdown
+
+The **Dropdown** widget renders a Bootstrap 5 dropdown menu component within the **Yii framework**.
+
+You can use it to create interactive dropdown menus with various item types such as links, buttons, headers, dividers,
+or plain text. This component is highly customizable, supporting directions, alignments, themes, and toggle variations.
+
+## Key Features
+- Supports multiple item types: links, buttons, headers, dividers, and text.
+- Customizable toggle button or link with variants, sizes, and split options.
+- Configurable menu alignment and direction using Bootstrap 5 classes.
+- Auto close behavior control (for example, close on inside/outside clicks).
+- Flexible HTML attribute and CSS class management.
+
+## Quick Start
+To get started, instantiate the **Dropdown** widget and add items to it. Here is a basic example:
+
+```php
+use Yiisoft\Bootstrap5\Dropdown;
+use Yiisoft\Bootstrap5\DropdownItem;
+
+echo Dropdown::widget()
+    ->togglerContent('Toggle Dropdown')
+    ->items(
+        DropdownItem::link('Action', '#'),
+        DropdownItem::link('Another action', '#'),
+        DropdownItem::divider(),
+        DropdownItem::link('Separated link', '#')
+    )
+    ->render();
+```
```

Comment:
Since widgets are primarily used within views, I'd provide examples for views:

```php
<?php
use Yiisoft\Bootstrap5\Dropdown;
use Yiisoft\Bootstrap5\DropdownItem;
?>

<?= Dropdown::widget()
    ->togglerContent('Toggle Dropdown')
    ->items(
        DropdownItem::link('Action', '#'),
        DropdownItem::link('Another action', '#'),
        DropdownItem::divider(),
        DropdownItem::link('Separated link', '#')
    )
?>
```

Reply:
fixed.